_scalingo_api/events.md

```diff
@@ -1771,6 +1771,114 @@ Example object:
   }
 }
 ```
+--- row ---
+
+* **Database Maintenance Planned**
+
+_When:_ A maintenance is scheduled on your database
+`type=database_maintenance_planned`
+
+{:.table}
+| field                         | type      | description                     |
+| ------------------------------| --------- | ------------------------------- |
+| maintenance_id                | string    | The related maintenance ID      |
+| addon_name                    | string    | The related addon name          |
+
+||| col |||
+
+Example object:
+
+```json
+{
+  "id":"62d13ad1a5b6ec0001e553c8",
+  "created_at":"2022-07-15T10:00:49.703Z",
+  "type":"database_maintenance_planned",
+  "app_id": "5343eccd646173000a140000",
+  "app_name": "appname",
+  "user": {
+    "username": "johndoe",
+    "email": "john@doe.com",
+    "id": "us-0e6d8e46-5cd0-42a4-acba-372b2be605ac"
+  },
```

Comment:
The API seems not returning this information. Eventually this should be the case.
In addition `type_data` will probably return more fields too

Reply:
For now those are the only fields the type data have. And yes for now the API does not contain any user data, so I removed it